src/index.html

```diff
@@ -11,7 +11,212 @@
+      </fieldset>
+      <fieldset class="group group-reg">
+        <legend>Registration</legend>
+        <label
+          class="label"
+          for="email"
+        >
+            E-mail:
+          <input
+            class="input"
+            name="email"
+            type="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </label>
+        <label
+          class="label"
+          for="password"
+        >
+            Password:
+          <input
+            class="input password"
+            type="password"
+            id="password"
+            name="password"
+            minlength="5"
+            maxlength="15"
+            required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group group-fact">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <label class="label">
+            Do you love cats?
+          <label
+            for="y"
+          >
+            <input
+              class="input"
+              type="radio"
+              id="y"
+              name="cats"
+            >
+              Yes
+          </label>
+          <label
+            for="n"
+          >
+            <input
+              class="input"
+              type="radio"
+              name="cats"
+              id="n"
+            >
+              No
+          </label>
+        </label>
+        <label
+          class="label"
+          for="color"
+        >
+            What is your favorite color?
+          <input
+            class="input"
+            name="color"
+            type="color"
+            id="color"
+          >
+        </label>
+        <label
+          class="label"
+          for="bed-time"
+        >
+            What time do you go to bed?
+          <input
+            class="input"
+            type="time"
+            id="bed-time"
+            step="2"
+            name="appt-time"
+          >
+        </label>
+        <label class="label">
+          What are your favorite brand of cars?
+          <select
+          name="choose-brand-of-cars"
+          multiple
+          >
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label class="label">
+          How do you rate our work?
+          <input
+            class="input"
+            type="range"
+            min="0"
+            value="0"
+            name="rate"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="group group-add">
+        <legend>Additional info:</legend>
+        <label
+          class="label"
+          for="comment"
+        >
+            Comments:
+          <textarea
+          name="comment"
+          id="comment"
+          cols="20"
+          rows="2"
+          autocomplete="off"
+          >
+          </textarea>
```

Comment:
[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) p.8
```suggestion
          <textarea
            name="comment"
            id="comment"
            cols="20"
            rows="2"
            autocomplete="off"
         ></textarea>
```